ftw/referencewidget/converter.py

```diff
@@ -49,3 +50,27 @@ def toFieldValue(self, value):
     def toWidgetValue(self, value):
         if value:
             return '/'.join(value.getPhysicalPath())
+
+
+class ReferenceDataListWithChoiceConverter(converter.BaseDataConverter):
+    """Converter of IList of IRelationChoices."""
+
+    adapts(IList, IReferenceWidget)
+
+    def toFieldValue(self, value):
+        if not value:
+            return []
+        elif isinstance(value, unicode):
```

Comment:
Hmm, I think the list handling is ok, but I would change it to:
```diff
-        elif isinstance(value, unicode):
+        elif isinstance(value, (unicode, str)):
```
just for safety reasons 😉